service/src/test/java/aero/minova/cas/service/SecurityServiceTests.java

```diff
@@ -461,8 +455,10 @@ void test_extractUserTokensGetEmptyStringBack() {
 		inputRow.addValue(new Value("codemonkey", null));
 		inputRow.addValue(new Value(true, null));
 		userGroups.add(inputRow);
-		List<String> resultList = testSubject.extractUserTokens(userGroups);
-		assertThat(resultList).hasSize(0);
+		List<String> resultList = securityService.extractUserTokens(userGroups);
+
+		assertThat(resultList)
```

Comment:
Was für ein Formatierer wird hier verwendet?
Das fällt an einigen Stellen auf.

Reply:
Es wurde `MINOVA-java.xml` in IntelliJ verwendet:
```
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<profiles version="11">
<profile kind="CodeFormatterProfile" name="MINOVA" version="11">
...
```

Reply:
Ich habe nur den Eindruck, dass die Minova-Format-Datei von Eclipse und IntelliJ unterschiedlich interpretiert wird.